api/reqs/tests/views_policies_tests.py

```diff
@@ -156,3 +156,31 @@ def test_nonpublic_reqs():
     assert policy.requirements.count() == 5
     assert response['results'][0]['relevant_reqs'] == 4
     assert response['results'][0]['total_reqs'] == 4
+
+
+@pytest.mark.django_db
+def test_omb_policy_id():
+    client = APIClient()
+    omb_policy_id = 'M-123-4'
+    path = "/policies/{0}.json".format(
+        omb_policy_id)
+    response = client.get(path).json()
+    assert response['detail'] == "Not found."
```

Comment:
What do you think about testing the response (prior to `.json()`)'s status code instead? Testing for a 404 is probably a bit more meaningful as we don't check `detail` in the frontend.